BanjoBotAssets/Exporters/UObjects/AccoladeExporter.cs

```diff
@@ -25,5 +25,15 @@ internal sealed class AccoladeExporter(IExporterContext services) : UObjectExpor
         protected override string Type => "Accolades";
 
         protected override bool InterestedInAsset(string name) => name.Contains("/AccoladeId_STW_", StringComparison.OrdinalIgnoreCase);
+
+        protected override Task<bool> ExportAssetAsync(UObject asset, NamedItemData itemData, Dictionary<ImageType, string> imagePaths)
+        {
+            if(asset.GetOrDefault<FScalableFloat>("XpRewardAmount") is FScalableFloat xpAmount)
+            {
+                //this should probably be moved to its own property in an ccolade-specific version of NamedItemData
```

Comment:
Indeed it should. :)

Reply:
A new NamedItemData subclass has been created for Accolades, in the same format as other NamedItemData subclasses, and it includes an AccoladeXP property to hold XP Reward Amounts